src/parser/jobs/brd/modules/Sidewinder.js

```diff
@@ -108,30 +207,121 @@ export default class Sidewinder extends Module {
 		if (!this._badCasts.length) {
 			return
 		}
+
+		let totalPotencyLost = 0
 		// Builds a list item for each incorrect cast
 		const items = this._badCasts.map(cast => {
 			const ability = getDataBy(ACTIONS, 'id', cast.abilityId)
-			if (cast.dotsApplied.length) {
-				return <List.Item key={cast.timestamp}>
-					<List.Content>
-						<Trans id="brd.sidewinder.list.one-dot">
-							{this._createTimelineButton(cast.timestamp)} <ActionLink {...ability}/> was cast with only <StatusLink {...getDataBy(STATUSES, 'id', cast.dotsApplied[0])}/> applied.
-						</Trans>
-					</List.Content>
-				</List.Item>
+			let singleTargetItem
+			let notBothDotsItem
+
+			if (!cast.hasBothDots) {
+				if (cast.dotsApplied) {
+					notBothDotsItem =
+							<Trans id="brd.sidewinder.list.one-dot">
+								<ActionLink {...ability}/> was cast with only one DoT applied to the target {cast.targetsHit > 1 ? 'and hit ' + cast.targetsHit + ' enemies' : ''}
```

Comment:
Likewise, can't be translated. Best way to check is to double-check the extracted message files - this particular message comes through as `<0/> was cast with only one DoT applied to the target {0}` - so that `{0}` is gobbling up the EN text.